<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAVSetting</class>
 <widget class="QWidget" name="DlgAVSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音视频设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hBoxTop">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="hBoxTitle">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hBoxSysBtn">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vBoxClient">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabAVSetting">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>音频</string>
        </attribute>
        <widget class="QComboBox" name="cmbMicList">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>351</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="cmbSpeakerList">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>351</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>15</y>
           <width>151</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>声音输入</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>65</y>
           <width>161</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>声音输出</string>
         </property>
        </widget>
        <widget class="QSlider" name="sliderMic">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>140</y>
           <width>100</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>204</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="sliderSpeaker">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>140</y>
           <width>100</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>204</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>调节</string>
         </property>
        </widget>
        <widget class="QLabel" name="micImageLabel">
         <property name="geometry">
          <rect>
           <x>33</x>
           <y>140</y>
           <width>21</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="soundImageLabel">
         <property name="geometry">
          <rect>
           <x>203</x>
           <y>140</y>
           <width>21</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>170</y>
           <width>241</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>自动调节麦克风音量</string>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>200</y>
           <width>311</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>提示：请对着麦克风说话，进行试听测试.</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>视频</string>
        </attribute>
        <widget class="QLabel" name="lblVideo">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>231</width>
           <height>211</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../Resources/setting/SeanVideoTestDefalutImage.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QComboBox" name="cmbCamList">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>90</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>10</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>预览</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>30</y>
           <width>131</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>我的本地图像</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_3">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>50</y>
           <width>151</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>对方看到的图像</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>70</y>
           <width>161</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>请选择你的视频设备</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>120</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>优先选项</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_4">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>140</y>
           <width>151</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>优先保证画面清晰</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_5">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>160</y>
           <width>151</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>优先保证视频流畅</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_6">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>190</y>
           <width>151</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>节省宽带模式</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>220</y>
           <width>81</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>画质调节</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>230</y>
           <width>221</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>视频组件启动成功!</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hBoxBottom">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
